src/components/widgets/Dropdown.js

```diff
@@ -64,7 +64,7 @@ class Dropdown extends Component {
       <div className={dropdownContainerClass} style={dropdownStyle}>
         <Select
           backspaceToRemoveMessage={backspaceToRemoveMessage}
-          placeholder={placeholder || _('select an option...')}
+          placeholder={placeholder || _('Select an Option')}
```

Comment:
I don't see these in the key files - can you collect them again?
FWIW it's best to do `npm run make:combined-translation-keys && npm run make:arrows` - the former is included in `make:lib` but not the latter, we should probably add it. We don't actually use `xx.js` from this repo right now, we only use the one created in batch, but eventually when we're using this editor in other contexts we probably will use the one here.